packages/fxa-settings/src/models/integrations/signin-signup-info.ts

```diff
@@ -95,9 +93,9 @@ export class SignInSignUpInfo extends ModelDataProvider {
   redirectTo: string | undefined;
 
   @IsOptional()
-  @IsBoolean()
+  @IsBooleanString()
   @bind(T.snakeCase)
-  returnOnError: boolean | undefined;
+  returnOnError: string | undefined;
```

Comment:
Is it annoying that the type is string? Maybe we should make it "true" | "false"?

Reply:
Ah yes, that's much better